src/Microsoft.AspNetCore.Routing/ParameterTransformer.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Routing
+{
+    public abstract class ParameterTransformer : IParameterPolicy
```

Comment:
I think this should probably be an interface, since you can be more than one class. Any good reason for this to be a class?